README.md

```diff
@@ -91,30 +91,27 @@ You may also need to install libcairo
 sudo apt-get install libcairo2 libcairo2-dev
 ```
 
-## Running
+## Input data
 
-Before you run, you'll need some data in the `data` directory.
-For convenience, there's a small test dataset available in the `testdata` directory.
-To use this data just copy over like `cp -r testdata data`.
-(See the data processing section below for more info on building custom data.)
+Olmsted uses [json-schema](https://json-schema.org/) to standardize input data. For a human-readable version of the schema, see [olmstedviz.org/schema.html](http://www.olmstedviz.org/schema.html) or view [schema.html](https://github.com/matsengrp/olmsted/blob/master/schema.html
+) on [htmlpreview.github.io](https://htmlpreview.github.io)
 
-you can now run Olmsted locally in development mode with `npm start localData`.
-Once that's ready, you can open a browser to [http://localhost:3999](http://localhost:3999/) to interact with the app.
+To parse input JSON files, use bin/process_data.py. E.g.:
 
+``` 
+./bin/process_data.py -i example_data/full_schema_input.json -o example_data/build_data -v -n inferred_naive
```

Comment:
a) `naive` is the default now
b) If you want to be explicit, maybe use something like `inferred_naive_name`, to make the point clearer

Reply:
For the example data, `inferred_naive` is the naive name, so using the default wouldn't work here. I can change the naive name for the example data if that is important?